pkgs/tools/backup/zfsbackup/default.nix

```diff
@@ -0,0 +1,25 @@
+# This file was generated by https://github.com/kamilchm/go2nix v1.3.0
+{ lib, buildGoPackage, fetchgit }:
+
+buildGoPackage rec {
+  pname = "zfsbackup";
+  version = "unstable-2019-03-05";
+  rev = "78fea6e99f0a5a4c8513d3a3d1d45fb6750cfddf";
+
+  goPackagePath = "github.com/someone1/zfsbackup-go";
+
+  src = fetchgit {
```

Comment:
Since the sources are on GitHub it should be `FetchFromGitHub` here instead of `fetchgit`.